backend/src/map_model.rs

```diff
@@ -834,24 +931,40 @@ impl FilterKind {
     }
 }
 
-#[derive(Clone, Copy, PartialEq)]
+#[derive(Clone, Debug, Copy, PartialEq, PartialOrd, Ord, Eq, Serialize, Deserialize)]
+pub enum TravelFlow {
+    BothWays,
+    OneWay(Direction),
+}
+impl TravelFlow {
```

Comment:
@dabreegster - this is kind of petty, so I wanted to check in. We were using a bool to represent `is_forwards` in a lot of places, even more with the new routing. 

To make it more self-documenting, I juggled things around, and renamed the "old" `Direction` -> `TravelFlow`.

Are you OK with this, or do you prefer the old way? I have a slight preference for the new way, but don't feel that strongly about it. If you're ok to move forward with this refactor, I'd like to proceed further and rename some fields/variables (from `direction(s)` -> `travel_flow(s)`) before marking as "ready for review".

Reply:
This new way is fine by me, feel free to finish renaming. The `is_forwards` bool was always kind of lazy/placeholder. In A/B Street, we had `DirectedRoadID(RoadID usize, bool forwards/backwards)` to make this cleaner.